src/modules/services.js

```diff
@@ -0,0 +1,120 @@
+import { createModule } from 'redux-modules';
+import { loop, Cmd, liftState } from 'redux-loop';
+import { getHost } from '../utils/network';
+import { log } from 'redux-modules-middleware';
+import { fetchConfig } from '../utils/fetchUtils';
+
+const endpoint = `${getHost()}/services`;
+
+const create = (params, onSuccess) =>
+  fetch(endpoint, { method: 'POST', body: JSON.stringify(params), headers: fetchConfig() })
+    .then(res => res.json())
+    .then(data => {
+      onSuccess(data)
+      return data
+    })
+
+const serializeForCreate = params => {
+  return {
+    // Update with services info
+    beds: params.bedsAvailable,
+    paid: params.price,
+    city: params.city,
+    length_of_stay: params.duration,
+    neighborhood: params.neighborhood,
+    housing_type: params.housingType,
+    has_animals: params.householdHasAnimals,
+    length_of_stay: params.duration,
+    notes: params.description,
+    child_friendly: params.childFriendly,
+    kid_notes: params.childNotes,
+    pets_accepted: params.petsAllowed,
+    pet_notes: params.petNotes,
+    contact_name: params.name,
+    phone_number: params.phoneNumber,
+    email_address: params.emailAddress,
+  };
+};
+
+const list = params =>
+  fetch(endpoint, { headers: fetchConfig() }).then(res => res.json());
+
+  // TO DO: services examples
+  const examplePayload = [
+  {
+    city: 'Ventura',
```

Comment:
Let's either remove this data, or replace the example data with keys that are more relevant to services.

Reply:
Removed

Also, will you show us on the doll where the gerbil touched you?